kv/txn_test.go

```diff
@@ -393,9 +393,9 @@ func TestUncertaintyMaxTimestampForwarding(t *testing.T) {
 		// node clock (which is ahead of keyFast as well). If the last part does
 		// not happen, the read of keyFast should fail (i.e. read nothing).
 		// There will be exactly one restart here.
-		call = client.Get(keySlow)
-		gr := call.Reply.(*proto.GetResponse)
-		if err = txn.Run(call); err != nil {
+		getCall := client.Get(keySlow)
+		gr := getCall.Reply
```

Comment:
Yeah, this is just removing the type assertion from the old test (which in turn had been mechanically transformed through a few iterations). I agree it would be better to not pull out .Reply until it's needed.